torch/overrides.py

```diff
@@ -1580,6 +1577,10 @@ def get_overridable_functions() -> Dict[Any, List[Callable]]:
                 continue
 
             if not callable(func) and hasattr(func, "__get__"):
+                index[func.__get__] = f"{namespace_str}.{func_name}.__get__"
```

Comment:
Should this be added only if we actually add this to overridable_funcs below?

Reply:
No, the name resolution and list of overridable funcs is intentionally different (although maybe it shouldn't be). Overridable funcs is not actually a complete list of all the functions that can show up in `__torch_function__`, and for name resolution I want as complete a list as possible